client/data/promote-post/types.ts

```diff
@@ -0,0 +1,58 @@
+export enum CampaignStatus {
```

Comment:
The status is updated via https://github.com/Automattic/wp-calypso/pull/76415/files#diff-3b65ffb0803f6f0a542a7b1373abcae3fa51454ed74898cc3fc6549e98d7ba78, so I guess this is the correct one.